lms/djangoapps/learner_dashboard/views.py

```diff
@@ -4,27 +4,14 @@
 
 from edxmako.shortcuts import render_to_response
 
-from lms.djangoapps.learner_dashboard.programs import ProgramsFragmentView, ProgramDetailsFragmentView
+from lms.djangoapps.learner_dashboard.programs import ProgramDetailsFragmentView
+from openedx.core.djangoapps.programs.views import program_listing as base_program_listing
```

Comment:
Can't find `views` module in the `openedx.core.djangoapps.programs`. Is it correct import?

Reply:
@cmltaWt0 It's correct, take a look: https://github.com/raccoongang/edx-platform/blob/drtwisted/fls-164-program-purchasing-feature/openedx/core/djangoapps/programs/views.py

Reply:
Oh. I'm sorry - it was added in this branch :) (I've tried to find it in lakeside-stage branch)